src/execution/execute.js

```diff
@@ -209,13 +209,16 @@ function buildExecutionContext(
     rawVariableValues || {}
   );
 
+  const executionPath = [];
```

Comment:
I don't believe this change is necessary. The value this returns is different from `info`, and an path is already created during initial execution on line 243